be/src/runtime/plan_fragment_executor.cpp

```diff
@@ -303,6 +303,8 @@ Status PlanFragmentExecutor::open_internal() {
     }
     RETURN_IF_ERROR(_sink->open(runtime_state()));
 
+    SET_CUR_TIME(ADD_TS(profile(), "ExecStartTime"));
```

Comment:
I think we should put this in PlanFragmentExecutor::open function. Then if open_internal return error we can get ExecEndTime